scout/server/static/custom_images.js

```diff
@@ -9,15 +17,24 @@
 		 .then(blob => {
 				 const img = document.createElement("img");
 				 img.src = URL.createObjectURL(blob);
+				 if (width > imgMaxWidth || height> imgMaxHeight) {
+				 		var resized_pixels = calculateAspectRatioFit(width, height, imgMaxWidth, imgMaxHeight);
```

Comment:
```suggestion
				 		const resized_pixels = calculateAspectRatioFit(width, height, imgMaxWidth, imgMaxHeight);
```

I would stick with the block scoped `let` and `const` of you don't have specific reasons not to. Here the value is not mutated, so `const` would be the correct one.